feature-libs/quote/components/summary/seller-edit/quote-summary-seller-edit.component.service.ts

```diff
@@ -149,32 +133,15 @@ export class QuoteSummarySellerEditComponentService {
     );
   }
 
-  /**
-   * Retrieves maximum number of decimal places. This supports validation, but it is not sufficient to do a complete validation,
-   * cases where the granted discount exceeds the total quote value by 1 are not covered (covered in OCC call).
-   * Still we want to inform the user as early as possible.
-   * Note that we assume currencies always come with 2 decimal places. In case this is not desired, this service can be overridden.
-   * @param quote - Quote
-   * @returns Maximum number of places, including 2 decimal places
-   */
-  getMaximumNumberOfTotalPlaces(quote: Quote): number {
-    const numberOfDecimalPlaces = 2;
-    const maximum = Math.max(
-      quote.totalPrice.value ?? 1,
-      quote.quoteDiscounts?.value ?? 1
-    );
-    return Math.floor(Math.log10(maximum)) + 1 + numberOfDecimalPlaces;
-  }
-
-  protected checkAndReportCurrencyIfMissing(
+  protected checkAndReportPercentageSignIfMissing(
     locale: string,
     formatter: Intl.NumberFormat,
-    currency?: string
+    percentageSign?: string
   ): LocalizationElements {
-    if (currency) {
-      return { locale, formatter, currencySymbol: currency };
+    if (percentageSign) {
+      return { locale, formatter, percentageSign: percentageSign };
     } else {
-      throw new Error('Currency must have symbol or ISO code');
+      throw new Error('No percentage sign present');
```

Comment:
Do we want to throw a console error or show a global message as well?

Reply:
I guess that's not needed, as such an error would be reflected in the console. Moreover that should not happen at runtime for this formatter (we request a percent formatter)